README.md

```diff
@@ -63,6 +64,12 @@ perform pool operations e.g pool provisioning.
 **Note:** It is not recommended to modify the CSPI CR and pool-manager in the running cluster unless you know what you are 
 trying to do. CSPC should be the only point of interaction.
 
+Once the CStor pool get provisioned successfully, admin/user can create PVC to provision csi CStor volumes, when a user
```

Comment:
n_p:
From control plane perspective and hence from SRE and users -- CStor Pool is not provisioned.
`CStorPoolCluster` is provisioned. CStor pools are the end result of CSPC provisioning. 
So I would prefer : 

`Once the CStorPoolCluster get provisioned`

Reply:
If we are mentioning like this at other places and my idea sounds good then we will fix it at all other places.

Reply:
n_p: 
`provision csi CStor volumes,` ---> `provision csi CStor volumes.`
Notice the comma changed to a period.